### PR TITLE
Add AWS ELB service account whitelist

### PR DESCRIPTION
**Pull request by jsoucheiron**
This PR adds ELB service accounts and makes sure S3 bucket rules don't trigger due to the inclusion of any of this accounts in a bucket policy

### CODE REVIEW

File: cfripper/model/utils.py [hunk 1]
```diff
@@ -101,3 +104,15 @@ def get_managed_policy(managed_policy_arn):
...
+def get_account_id_from_arn(arn) -> Optional[str]:
+    match = REGEX_ARN.match(arn)
+    if match:
+        return match.group(3)
+
+
+def get_account_id_from_iam_arn(arn) -> Optional[str]:
+    match = REGEX_IAM_ARN.match(arn)
+    if match:
+        return match.group(1)
```
Comment (IgnacioRV) [inline cfripper/model/utils.py]: In principals, specifying only the account id in an arn is equivalent to `arn:aws:iam::<account_id>:root`, this case should also be considered

https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_principal.html

File: cfripper/model/utils.py [hunk 2]
```diff
@@ -101,3 +104,21 @@ def get_managed_policy(managed_policy_arn):
             }
         )
     return None
+
+
+def get_account_id_from_arn(arn) -> Optional[str]:
```
Comment (ocrawford555) [inline cfripper/model/utils.py]: Add type information to `arn` parameter here (looks like a `str`)
Same for function below as well :)

File: cfripper/model/utils.py [hunk 3]
```diff
@@ -101,3 +104,21 @@ def get_managed_policy(managed_policy_arn):
...
+    match = REGEX_ARN.match(arn)
+    if match:
+        return match.group(3)
+
+
+def get_account_id_from_iam_arn(arn) -> Optional[str]:
```
Comment (IgnacioRV) [inline cfripper/model/utils.py]: This function isn't used 👀

File: cfripper/model/utils.py (hunk 2 above)
Comment (jsoucheiron) [reply cfripper/model/utils.py]: done

File: cfripper/model/utils.py (hunk 3 above)
Comment (jsoucheiron) [reply cfripper/model/utils.py]: True, now the get_account_id_from_arn is not used, but it's ok. It's useful anyway.

**Pull request merged**